pkgs/applications/window-managers/qtile/default.nix

```diff
@@ -41,7 +43,7 @@ python37Packages.buildPythonApplication rec {
       --run 'export QTILE_SAVED_PATH=$PATH'
   '';
 
-  doCheck = false; # Requires X server.
+  doCheck = false; # Requires X server #TODO Would this not work with the usual nixos test infrastructure or xvfb?
```

Comment:
yes, but you would need to write a nixos test for that. which would be not in scope for this package. Bot you could at a test to nixos/tests/

Reply:
Yes, but I think the TODO should be left in so if someone else runs across it they can decide to do something about it.

Reply:
well currently it poses a question and I answered it. It would still be nice to fix the tests in here or make them not depend on X upstream. So I guess your point is somewhat orthogonal to the tests being deactivated here.

Reply:
I made it a statement in 3aaf3f1.

Reply:
Ok whatever, I think I'm going to try to get the tests enabled.

Reply:
ok, should I merge this and you will open a new PR for the tests?